apps/backend/.env

```
@@ -0,0 +1,38 @@
+PERSONNALISATION_QUESTIONS_SHEET_ID=encrypted:BN2RrN7CB/Ygje/4lJ9wh4hAiRnmsrek6Fv4jcMi7XqfXgLYPW8cRQHo30oCMlLji7cy6WjW9KrjBafCfPdEco4acEBUi6SAUfKVcvIIQ4h/1x11tIlRsww4oIRYv3V8bkzgXM2SxJhgclDJ1rWYe7byyQGkdIAc6NmLjGHZNoi4caO+7EKLOv/lne92
+
+# Host et port du serveur Redis utilisé pour la file d'attente des tâches asynchrones
+QUEUE_REDIS_HOST=localhost
+QUEUE_REDIS_PORT=6379
+
+# URLs for `local` env
+SUPABASE_DATABASE_URL=postgresql://postgres:postgres@localhost:54322/postgres
+SUPABASE_URL=http://127.0.0.1:54321
+SUPABASE_SERVICE_ROLE_KEY=encrypted:BJY4pCvYdwDuPxrw0fXu5N/kzRvGsEbM38M0XeDCJY51vT4SAvQ753IMZCZab5KxIX/w6BfDDn9FSCVOGI3CMKQItOch85tR+dVehasF6danl+GHn5Y+GnI8PjSognZJJYrlh4FLVYRsu3HMOizHkzep4ar0aH8F99g9zggE4+dMVy+L8OvyS4u33a4YCrgTdmBqsfK0wQC3Bedms4b++B3Tuqxe2vHAZIjzOkuZ1a4OTSVt5YczqFE2HVWsFEgnlt7BFvORycfE0WX9OI+QIwuwq/S5XaJ8UfoFn8+HEoXsB/H2gHcOniRcp606mXt4LqAZ/T3AamNwq3A8mm+DsfAy6aRg
+SUPABASE_ANON_KEY=encrypted:BIBZaWet0tO56fs5u16Sprjaub30uAeVNkbmFvgs8jG59ORqC373xWs/mVXcq9YxIqaphDjyhoENBzLpiyhV3oW89DWLlU9XURtqzSoHWODtKQ/OY2i2Se7wKtDcQPL8kJahDB5B3oMLnyJvnJOUi2gNFsfS8qMJx2yLXjVIohelS5aJFpnuL8AUo/H8hCbxzONZA8nTmUqxmxYvIOKiLE3EoFS4gUGQfcdYruSXhXcKDxk/5KUzd72Yd8rfpbQarDvm3dGZk6eLLbvOfHISqN+sraQZ83nJCJ/F2837X/AhCOAOJ5dnsiFPMD7C5tdon6uM6dz6Z42L+g==
+SUPABASE_JWT_SECRET=super-secret-jwt-token-with-at-least-32-characters-long
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)